### PR TITLE
Fix code samples

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -225,11 +223,11 @@ search_parameter_guide_filter_1: |-
     filter: 'title = Nightshift'
   })
 search_parameter_guide_filter_2: |-
-  client.index.search('n', {
+  client.index.search('shifu', {
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:226]:
```suggestion
  client.index('movies').search('shifu', {
```
At the same time, do we take the opportunity to put the name of the index?

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -324,10 +321,10 @@ faceted_search_update_settings_1: |-
     'director',
     'genres'
   ])
-# faceted_search_facet_filters_1: |-
-#   client.index.search('thriller', {
-#     facetFilters: [['genres:Horror', 'genres:Mystery'], 'director:Jordan Peele']
-#   })
+faceted_search_filter_1: |-
+  client.index.search('thriller', {
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:325]:
```suggestion
  client.index('movies').search('thriller', {
```
Here too?

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -339,10 +336,10 @@ faceted_search_walkthrough_filterable_attributes_1: |-
...
-#   client.index.search('thriller', {
-#     facetFilters: [['genres:Horror', 'genres:Mystery'], 'director:Jordan Peele']
-#   })
+faceted_search_walkthrough_filter_1: |-
+  client.index.search('thriller', {
+    filter: [['genres = Horror', 'genres = Mystery'], 'director = "Jordan Peele"']
+  })
 faceted_search_walkthrough_facets_distribution_1: |-
   client.index('movies').search('Batman', {
     facetsDistribution: ['genres']
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:345]: You didn't add `phrase_search_1:`

**Comment by alallema** [comment]
If you decide to add the index name, it is also missing in all the filter guides.

File: .code-samples.meilisearch.yaml (hunk 1 above)
Comment (curquiza) [reply .code-samples.meilisearch.yaml:226]: Completely!! Thanks!

File: .code-samples.meilisearch.yaml (hunk 3 above)
Comment (curquiza) [reply .code-samples.meilisearch.yaml:345]: Thanks, I completely missed this one :D

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -211,25 +209,25 @@ search_parameter_guide_crop_1: |-
...
   })
 search_parameter_guide_highlight_1: |-
-  client.index('movies').search('shifu', {
+  client.index('movies').search('winter feast', {
     attributesToHighlight: ['overview']
   })
 search_parameter_guide_matches_1: |-
-  client.index.search('n', {
+  client.index('movies').search('n', {
     filter: 'title="Kung Fu Panda"',
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:217]:
```suggestion
    filter: 'title="winter feast"',
```

**Review by alallema — COMMENTED** [review]
`update_filterable_attributes_1` and `filtering_guide_1` still have `index.` instead of `client.index('movies')`

**Review by alallema — APPROVED** [review]
LGTM!
Thanks @curquiza

**Comment by curquiza** [comment]
Thanks for reviewing!

**Pull request merged**